test/test-frames/sen_pollutherm.norm.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<MBusData>
+
+    <SlaveInformation>
+        <Id>21050076</Id>
+        <Manufacturer>SPX</Manufacturer>
+        <Version>49</Version>
+        <ProductName>Sensus PolluTherm</ProductName>
+        <Medium>Heat: Outlet</Medium>
+        <AccessNumber>81</AccessNumber>
+        <Status>00</Status>
+        <Signature>0000</Signature>
+    </SlaveInformation>
+
+    <DataRecord id="0">
+        <Function>Instantaneous value</Function>
+        <StorageNumber>0</StorageNumber>
+        <Unit>Wh</Unit>
+        <Quantity>Energy</Quantity>
+        <Value>8640000.000000</Value>
+    </DataRecord>
+
+    <DataRecord id="1">
+        <Function>Instantaneous value</Function>
+        <StorageNumber>0</StorageNumber>
+        <Unit>m^3</Unit>
+        <Quantity>Volume</Quantity>
+        <Value>7998.920000</Value>
+    </DataRecord>
+
+    <DataRecord id="2">
+    </DataRecord>
```

Comment:
See comment https://github.com/rscada/libmbus/pull/168#issuecomment-651723887 .

This is probably where this is printed.

```
mbus_vif_unit_normalize: Unknown VIF 0x07B
mbus_vib_unit_normalize: Error mbus_vif_unit_normalize
mbus_parse_variable_record: problem with mbus_vib_unit_normalize
```

Reply:
Perhaps skip checking in this file while this issue remains?